migrations/0043-add-discord-and-google-calendar-event-fields.ts

```diff
@@ -0,0 +1,33 @@
+import { MigrationInterface, QueryRunner, TableColumn } from 'typeorm';
+
+const TABLE_NAME = 'Events';
+
+export class AddDiscordAndGoogleCalendarEventFields1711750534274 implements MigrationInterface {
```

Comment:
Small nit: instead of Field, use column in the name of the migration and filename